justfile

```diff
@@ -1,3 +1,7 @@
 container:
     docker build -t penumbra_dapp .
     docker run -p 9012:9012 -it penumbra_dapp
+
+dev-shell:
+    docker build -t penumbra_dapp .
+    docker run -p 9012:9012 -v $PWD:/usr/src/app -it penumbra_dapp bash
```

Comment:
Technically this bit is extraneous, but it's useful for me when debugging locally. :upside_down_face: